### PR TITLE
Update release, dependencies and minor refactor

### PR DESCRIPTION
**Pull request by omerzi**
- [x] All [tests](https://github.com/jfrog/frogbot#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] This pull request is on the dev branch.
- [ ] I used gofmt for formatting the code before submitting the pull request.
---

### CODE REVIEW

File: commands/utils/consts.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ const (
 	JFrogUrlEnv            = "JF_URL"
 	jfrogXrayUrlEnv        = "JF_XRAY_URL"
 	jfrogArtifactoryUrlEnv = "JF_ARTIFACTORY_URL"
-	jfrogRemoteRepo        = "JF_FROGBOT_REPO"
+	jfrogReleasesRepo      = "JF_RELEASES_REPO"
```
Comment (eyalbe4) [inline commands/utils/consts.go:36]: To be compatible with the other varoables defined here - 
jfrogReleasesRepo --> jfrogReleasesRepoEnv

File: commands/utils/params.go [hunk 2]
```diff
@@ -29,13 +29,29 @@ var errFrogbotConfigNotFound = fmt.Errorf("%s wasn't found in the Frogbot direct
...
 type FrogbotUtils struct {
-	ConfigAggregator *FrogbotConfigAggregator
+	ConfigAggregator FrogbotConfigAggregator
 	ServerDetails    *coreconfig.ServerDetails
 	Client           vcsclient.VcsClient
 }
 
 type FrogbotConfigAggregator []FrogbotRepoConfig
 
+// UnmarshalYaml uses the yaml.Unmarshaler interface to parse the yamlContent, and then sets default values if it wasn't supplied by the user.
```
Comment (eyalbe4) [inline commands/utils/params.go:39]: ...if it wasn't supplied... --> ...if they weren't set...

File: commands/utils/params.go [hunk 3]
```diff
@@ -29,13 +29,29 @@ var errFrogbotConfigNotFound = fmt.Errorf("%s wasn't found in the Frogbot direct
...
+func (fca FrogbotConfigAggregator) UnmarshalYaml(yamlContent []byte) (result FrogbotConfigAggregator, err error) {
```
Comment (eyalbe4) [inline commands/utils/params.go:40]: Let's add a unit test for this function, which includes all the default value cases.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: commands/utils/consts.go (hunk 1 above)
Comment (omerzi) [reply commands/utils/consts.go]: done

File: commands/utils/params.go (hunk 2 above)
Comment (omerzi) [reply commands/utils/params.go]: done

File: commands/utils/params.go (hunk 3 above)
Comment (omerzi) [reply commands/utils/params.go:40]: done

**Pull request merged**